app/views/dashboards/_table_content.html.erb

```diff
@@ -42,9 +42,20 @@
             <% end %>
           <% end %>
 
-          <td><%= link_to call_glyphicon, "#call-#{patient.primary_phone_display}", data: {toggle: "modal"} %></td>
+      <td>
+        <%= link_to call_glyphicon, "#call-#{patient.primary_phone_display}", class:"call_button",
+        data: {
+          patientname: patient.name,
```

Comment:
Yeah, sounds like we're in agreement here. I think here's the path to take:

* Add `:new` to the `resources :calls` array in `routes.rb`, and a `CallsController#new` method to match
* Fire up a new file `app/views/calls/new.js.erb` which renders the `_new_call` partial (pretty much what the coffeescript is doing, just with rails `escape_javascript(render ...)` helper deal)
* Have the phone icon link trigger that controller action. (We do something similar with `display_reached_patient_link` in `CallsHelper`.

I think from there it's going to be stuff on the order of making sure the js still works (modal clickaway, being able to create new calls via the link, etc).

For ref, we're calling `notes#create` here -- https://github.com/colinxfleming/dcaf_case_management/blob/master/app/views/patients/_notes.html.erb#L15 -- and this is the block that's telling rails to look for `create.js.erb`.

Thanks again, @tingaloo ! Feel like I'm picking up a decent amount of js just by reading your PRs.